api/config/config_test.json

```diff
@@ -62,5 +62,8 @@
   },
   "cohortbuilder": {
     "enableListSearch": false
+  },
+  "featureFlags": {
+    "useBillingProjectBuffer": true
```

Comment:
I don't think we're ready to enable this yet. There's a bunch of free tier billing project + cluster + UI tickets which would need to be resolved first. I will try to create a ticket which maps out all the dependencies.